package.json

```diff
@@ -28,6 +28,7 @@
     "mochawesome": "^7.1.3"
   },
   "dependencies": {
-    "axios": "^0.27.2"
+    "axios": "^0.27.2",
+    "dotenv": "^16.0.1"
```

Comment:
Why is this dependency?

Reply:
Which dependency? dotenv?

Reply:
Yes

Reply:
I had to install it in order to use the .env file to store the Access Token

Reply:
I see